bin/kubectl-crossplane-package-init

```diff
@@ -188,25 +188,25 @@ RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o manager
 # Alpine is used instead of distroless because the stack manager expects things like `cp` to exist
 FROM alpine:3.7
 WORKDIR /
-COPY stack-package /
+COPY package /
 COPY --from=builder /workspace/manager .
 ENTRYPOINT ["/manager"]
 EOF
-  echo 'Created stack.Dockerfile' >&2
+  echo 'Created package.Dockerfile' >&2
 
-  cat > stack.env <<EOF
+  cat > package.env <<EOF
 # Image URL to use all building/pushing image targets
-STACK_IMG ?= ${STACK_NAME}:latest
+PACKAGE_IMG ?= ${PACKAGE_NAME}:latest
 
 CRD_DIR ?= config/crd/bases
-STACK_PACKAGE_REGISTRY_SOURCE ?= config/stack/manifests
-LOCAL_OVERRIDES_DIR ?= config/stack/overrides
-CONFIG_SAMPLES_DIR ?= config/stack/samples
+PACKAGE_REGISTRY_SOURCE ?= config/package/manifests
```

Comment:
did you walk through the quick start all the way to do an init and a build to make sure that flow works?

how about for existing repos that were scaffolded out by this `init` command? is their repo structure going to be updated too?

Reply:
Yep I tested this out with a new repo. However, I don't think any of the crossplane repos currently use this full setup. Providers do this by hand and stacks / apps only use build and publish.